src/components/examples/ExamplesList.module.css

```diff
@@ -42,7 +42,8 @@
 }
 
 .item {
-  flex-basis: calc(100% / 5);
+  flex: 0 0 calc(25% - 2 * var(--gutter)); 
```

Comment:
The fix works, could you research if you could scale the container element instead?